v1.22/Dockerfile.amd64

```
@@ -0,0 +1,35 @@
+FROM alpine:3.13 AS fetcher
+LABEL maintainer="support@kubermatic.com"
+
+ARG VERSION
+ARG ARCH=amd64
+ARG CHECKSUM=091393bad069ee8bc31798e2c91b86cca32a9a16318fb8a373ae381a47b180987ae1adebefefe50c1bf144e17e7cd0af9f499aad1986b216f2485ff59514edbe
+
+RUN apk add curl && \
+    curl -Lo node.tar.gz https://dl.k8s.io/${VERSION}/kubernetes-node-linux-${ARCH}.tar.gz && \
+    echo "${CHECKSUM}  node.tar.gz" | sha512sum -c && \
+    tar xzf node.tar.gz kubernetes/node/bin/kubectl kubernetes/node/bin/kubelet
+
+FROM eu.gcr.io/k8s-artifacts-prod/build-image/debian-iptables:buster-v1.5.0
```

Nit: we should use `k8s.gcr.io`.

```suggestion
FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
```

---

Let's do that in a dedicated PR for all vesions